tasks/friends-app.md

```diff
@@ -0,0 +1,61 @@
+[![MIT Licensed][icon-mit]][license]
+[![Awesome][icon-awesome]][awesome]
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+[![Telegram][icon-chat]][chat]
+
+# Friends App
+
+When you're done with your challenge push your code to github repo and publish
+it using `gh-pages`. _Add link to it to your main frontend tasks repo._
+
+If you honestly finished all the previous steps than go ahead and share it with
+others - send a message in [course channel](chat) with the link to you repo and the words "Weather APP - finished" and add link to repo with code of the
+website.
+
+## Friends app
+
+Create a tiny social friends' search page with users cards, search, sorting and filtering of them by age, name or whatewer you want with [Random User API](https://randomuser.me) :)
+![image](https://i.imgur.com/5tcpqcY.png).
+
+### As a user, I would like to:
+
+* See list of my mock friends :)
+* can search them by typing name in search input and see immediately filtering on the page
+* sort cards by name/age in A-Z/Z-A ways
+* filter friends by age, by name, whatever you want else options
+
+- [ ] you can use all `Array.methods` which you know :) for instance - Array.sort (text sort, number sort) `/filter/slice/splice/shift/map/reduce`
+- [ ] you'll try to figure out with request/response data incapsulation and try callbacks' logic in action
```

Comment:
I think here some problems with grammar. What are you trying to say?

Reply:
I tried to improve this text but if you want to suggest some other changes please write them here more descriptive and I'll add all of them to this PR) thank you :)